src/ck-core/charm.h

```diff
@@ -405,9 +405,9 @@ extern void CkChareExtSend(int onPE, void *objPtr, int epIdx, char *msg, int msg
 extern void CkChareExtSend_multi(int onPE, void *objPtr, int epIdx, int num_bufs, char **bufs, int *buf_sizes);
 /// Send msg to group with ID 'gid'. if pe == -1, msg will be broadcasted, else
 /// it will go to the group instance in that PE
-extern void CkGroupExtSend(int gid, int pe, int epIdx, char *msg, int msgSize);
+extern void CkGroupExtSend(int gid, int npes, const int *pes, int epIdx, char *msg, int msgSize);
```

Comment:
Small suggestion: if it is an array, it is sometimes a bit clearer to write as `const int pes[]`.